woocommerce-gateway-ebanx/gateways/class-wc-ebanx-credit-card-gateway.php

```diff
@@ -177,15 +177,16 @@ public function checkout_assets()
 	 */
 	protected function request_data($order)
 	{
-		if (empty($_POST['ebanx_token']) ||
-			empty($_POST['ebanx_masked_card_number']) ||
-			empty($_POST['ebanx_brand']) ||
-			empty($_POST['ebanx_billing_cvv'])
+		
+		if (!WC_EBANX_Request::has('ebanx_token') ||
+			!WC_EBANX_Request::has('ebanx_masked_card_number') ||
+			!WC_EBANX_Request::has('ebanx_brand') ||
+			!WC_EBANX_Request::has('ebanx_billing_cvv')
```

Comment:
`empty($_POST)` should be replaced by `empty(WC_EBANX_Request::read($key, null))`